app/assets/stylesheets/application.scss

```diff
@@ -40,17 +47,113 @@ body {
   padding: 0 15px;
 }
 
-input[type=text].postfix.round,
-input[type=email].postfix.round {
-  border-radius: 1000px 0 0 1000px;
-  padding: 0;
+.background {
+  -webkit-background-size: contain;
+  -moz-background-size: contain;
+  -o-background-size: contain;
+  background: url(asset-path("footer-eight.jpg")) no-repeat;
+  background-size: contain;
+  background-position: center bottom;
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 720px;
+  overflow: auto;
+}
+
+.frontpage-form {
+  input[type=text] {
+    background: transparent;
+    border: none;
+    border-radius: 1000px 0 0 1000px;
+    color: rgba(101, 247, 255, 0.75);
+    height: 60px;
+    line-height: 60px;
+    padding: 0;
+  }
+
+  input[type=submit], .button {
+    background-color: #000DBA;
+    border-color: #007095;
+    border-radius: 1000px;
+    color: #6FFDFF;
```

Comment:
Color literals like `#6FFDFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#6FFDFF` should be written as `#6ffdff`